src/components/signin/SigninInfo.js

```diff
@@ -0,0 +1,127 @@
+// @flow
+import React from 'react'
+import { Image, ScrollView, View } from 'react-native'
+import NavBar from '../appNavigation/NavBar'
+import Section from '../common/layout/Section'
+import Circle from '../common/view/Circle'
+import Wrapper from '../common/layout/Wrapper'
+import Text from '../common/view/Text'
+
+import { withStyles } from '../../lib/styles'
+import illustration from '../../assets/Signin/illustration.svg'
+import normalize from '../../lib/utils/normalizeText'
+type Props = {
+  navigation: any,
+  screenProps: {
+    push: Function,
+  },
+  styles: any,
+}
+
+Image.prefetch(illustration)
+
+class Signin extends React.Component<Props> {
+  handleSignIn = () => {
+    this.props.navigation.navigate('Auth')
+  }
+
+  handleRecover = () => {
+    this.props.navigation.navigate('Recover')
+  }
+
+  render() {
+    const { styles } = this.props
+    return (
+      <View style={{ flexGrow: 1 }}>
+        <NavBar goBack={this.handleSignIn} title={'SIGN IN'} />
+        <ScrollView contentContainerStyle={styles.scrollableContainer}>
+          <View style={styles.contentContainer}>
+            <Wrapper backgroundColor="#fff" style={styles.mainWrapper}>
+              <Image source={illustration} style={styles.illustration} resizeMode="contain" />
+              <Section.Stack grow justifyContent="flex-start">
+                <Section.Row alignItems="center" justifyContent="center" style={styles.row}>
+                  <View style={styles.bottomContainer}>
+                    <Text fontWeight="medium" fontSize={normalize(22)} fontFamily="Roboto">
```

Comment:
The text component normalizes no need to normalize

Reply:
done [commit](https://github.com/GoodDollar/GoodDAPP/pull/517/commits/6e9e28de5d885a7c2fc26ee598452e51675d5866)